eth-contracts/.eslintignore

```diff
@@ -1,2 +1 @@
-test/MockSpv.test.js
-test/TokenLocker.test.js
+test/*.test.js
```

Comment:
why exclude this files? you can use tools to format them automaticlly.

Reply:
One of the trouble issues is that `ethers` is built into the Buidler testing framework so there is no need to import the library separately in the test files, and eslint checks will report errors, which I haven't found the rule to fix

Reply:
Add comment to skip the specific line, not the entire file.

ref: 
- https://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line
- https://eslint.org/docs/user-guide/configuring.html#configuring-rules

Reply:
under buidler, test.js is really special,  if using rules that you mentioned, comment will be as numerous as coding. I'll try to find some other way